src/parser/jobs/sch/modules/Aetherflow.js

```diff
@@ -189,6 +189,13 @@ export default class Aetherflow extends Module {
 							wasted = EXTRA_AETHERFLOWS - debit || 0
 							totalWasted += wasted
 						}
+
+						if (!Array.isArray(timestamp)) {
+							// I mean, they should be doing more than one AF cast per stack
+							// but who am I to judge?
+							timestamp = [timestamp]
+						}
+
 						return <Table.Row key={timestamp}>
```

Comment:
might be soem warnings on this, not sure if react is super happy about getting an array as a key. might be ok i guess.

will def re-render every time though.